ToDoApp/ToDoApp/CreateTaskView+ViewModel.swift

```diff
@@ -0,0 +1,22 @@
+//
+//  CreateTaskView+ModelView.swift
+//  ToDoApp
+//
+//  Created by Trong Hieu Vuong on 2022-12-02.
+//
+
+import Foundation
+
+extension CreateTaskView {
+    @MainActor class CreateTaskViewModel: ObservableObject {
```

Comment:
This has been fixed.

Reply:
The comment above was given regarding the class name, not the file name, let me describe it in more detail.
We can pick two options from naming conventions for the ViewModel:
1. We can name the file CreateTaskView+ViewModel. It will have the ViewModel class defined inside the extension like this: 
```
// CreateTaskView+ViewModel.swift

extension CreateTaskView {
    class ViewModel {
        //...
    }
}
```
2. We can name the file CreateTaskViewModel. And the class would be defined in the file scope instead inside of the extension, like this: 
```
// CreateTaskViewModel.swift

class CreateTaskViewModel: ObservableObject {
    // ...
}
```

for the first option why the class name is simply `ViewModel`? that's for the reason it is defined inside the `CreateTaskView` extension and if this class will be used outside it will have the following usage:
```
let viewModel: CreateTaskView.CreateTaskViewModel = // ... 
```
that's why there is no need to duplicate the `CreateTask` prefix since it is already there coming from the type name in which ViewModel is defined